lib/matplotlib/backends/backend_ps.py

```diff
@@ -1,9 +1,6 @@
 """
 A PostScript backend, which can produce both PostScript .ps and .eps
 """
-import six
-from six.moves import StringIO
```

Comment:
Do we not use `StringIO` in this module?

Reply:
Not anymore - it was only used in a `isinstance` check in a Python-2-only branch.